src/modules/add-remove.js

```diff
@@ -1,9 +1,14 @@
 import NewTask from './createTask.js';
-import discard from './images/bin.svg';
+import discard from '../images/bin.svg';
+import {
+  markItemAsComplete, deleteAllTasks,
+} from './interactive-page.js';
 
 const taskContainer = document.querySelector('.task-container');
 const addNewTaskInput = document.getElementById('add-new-task');
 const enterBtn = document.querySelector('.enter');
```

Comment:
- [ ] [OPTIONAL] In your add new task function, I see that you are adding a `click` event listener to the `button` to add an item. It would be a better user experience if they can add by clicking on the `enter` button.